testproject/Assets/Secure/generate.sh

```diff
@@ -0,0 +1,6 @@
+openssl genrsa -out clientPrivateKeyForRootCA.pem 2048
```

Comment:
is this file checked in intentionally?

Reply:
Yes. This is part of `testproject`. It could use a bit of documentation. But, yeah, it is meant as an example of how one could generate their certificate. Use just internally, every developer that wants to work on this will need to generate their own cert.

Reply:
No, it's fine if you say so. I was just double-checking.